### PR TITLE
EDGSITES-373 Unsupported connectors removed, cleanup of existing docs

### PR DESCRIPTION
**Pull request by jakub-jarek**

### CODE REVIEW

File: src/guides/v7/sites_frameworks/legacy_frameworks.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Legacy Framework Support
+---
+
+As of v7.6.0, {{ PRODUCT }} has dropped the previously-supported frameworks listed below. For current framework support, refer to the [supported frameworks guide](/guides/sites_frameworks/getting_started).
```
Comment (jsernaque) [inline src/guides/v7/sites_frameworks/legacy_frameworks.md:5]:
```suggestion
{{ PRODUCT }} v7.5.0 and below integrates the frameworks listed below through framework-specific connectors. These connectors are unsupported for use with {{ PRODUCT }} v7.6.0.
```

File: src/guides/v7/sites_frameworks/legacy_frameworks.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Legacy Frameworks {/*legacy-frameworks*/}
+
+The following frameworks are supported by {{ PRODUCT }} v7.5.0 and earlier:
```
Comment (jsernaque) [inline src/guides/v7/sites_frameworks/legacy_frameworks.md:9]:
```suggestion
<Callout type="important">

  If your project uses one of these frameworks, then we recommend integrating it through our generic [Node.js connector](/guides/sites_frameworks/getting_started/nodejs_connector) instead.

</Callout>
```

File: src/guides/v7/sites_frameworks/legacy_frameworks.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+<V7LegacyFrameworks />
+
+If you wish to use a later version of {{ PRODUCT }} or your framework is not listed, we recommend implementing the generic [Node.js connector](/guides/sites_frameworks/getting_started/nodejs_connector), which enables integration with {{ PRODUCT }} for various frameworks, irrespective of the version.
```
Comment (jsernaque) [inline src/guides/v7/sites_frameworks/legacy_frameworks.md:13]:
```suggestion
[View additional frameworks.](/guides/sites_frameworks/getting_started)
```

File: src/templates/sites_connector_deprecated.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<Callout type="warning">
+
+{{ PRODUCT }} v7.6.0 has deprecated support for this framework and any associated connector. To use this framework, you must use {{ PRODUCT }} v7.5.0 or earlier. Optionally, you may configure the universal [Node.js connector](/guides/sites_frameworks/getting_started/nodejs_connector) to use this framework.
```
Comment (jsernaque) [inline src/templates/sites_connector_deprecated.md:3]: Did we also deprecate support for the framework?

File: src/guides/v7/sites_frameworks/legacy_frameworks.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: Legacy Framework Support
+---
+
+As of v7.6.0, {{ PRODUCT }} has dropped the previously-supported frameworks listed below. For current framework support, refer to the [supported frameworks guide](/guides/sites_frameworks/getting_started).
+
+## Legacy Frameworks {/*legacy-frameworks*/}
```
Comment (tristanlee85) [inline src/guides/v7/sites_frameworks/legacy_frameworks.md:7]:
```suggestion
```

**Review by jsernaque — APPROVED** [review]
LGTM

**Pull request merged**